tests/wpt/metadata-css/selectors-3_dev/html/css3-selectors-lang-005.htm.ini

```diff
@@ -1,5 +1,5 @@
 [css3-selectors-lang-005.htm]
   type: testharness
   [A :lang value will match a lang attribute value when the latter contains additional subtags.]
-    expected: FAIL
+    expected: PASS
```

Comment:
You can simply remove this file. By default, tests are expected to PASS, the presence of an ini file is to explicitly mark them as either FAIL or TIMEOUT.